src/mds/mdstypes.h

```diff
@@ -1020,4 +1020,22 @@ inline bool operator==(const MDSCacheObjectInfo& l, const MDSCacheObjectInfo& r)
 }
 WRITE_CLASS_ENCODER(MDSCacheObjectInfo)
 
+struct EstimatedReplayTime {
+  EstimatedReplayTime(double percent_complete, std::chrono::seconds estimated_time,
+		      std::chrono::seconds elapsed_time)
+    : percent_complete(percent_complete),
+      estimated_time(estimated_time),
+      elapsed_time(elapsed_time) {
+  }
```

Comment:
I assume we have a constructor to satisfy Clang but it's not necessary. When constructing do:

`EstimatedReplayTime{percent, est_time, elapsed_time}` instead.